### PR TITLE
Add Inazuma Eleven GO: Light and Shadow

### PR DESCRIPTION
**Pull request by maplesuf**
Adds Inazuma Eleven GO: Light and Shadow.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/inazuma-eleven-go-light-and-shadow/game.dat [hunk 1]
```diff
@@ -0,0 +1,24 @@
+title = "Inazuma Eleven GO: Light and Shadow"
+description = "Inazuma Eleven GO is a role-playing and sports video game for the Nintendo 3DS, developed and published by Level-5. It's the first game of the Inazuma Eleven GO series and the fourth installment of the Inazuma Eleven games. There are two versions of the game, Light and Shadow."
+needs_system_files = false
+needs_shared_font = true
+
+[[releases]]
+title = "000400000006D400"
+region = "jpn"
+release_date = "2011-01-15"
```
Comment (Leo626) [inline games/inazuma-eleven-go-light-and-shadow/game.dat]: Wrong release date.

> JP: December 15, 2011

File: games/inazuma-eleven-go-light-and-shadow/game.dat [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+[[releases]]
+title = "000400000006D300"
+region = "jpn"
+release_date = "2011-01-15"
```
Comment (Leo626) [inline games/inazuma-eleven-go-light-and-shadow/game.dat]: Ditto.

**Comment by Leo626** [comment]
* Need to create the wiki entry, See [contributing guide](https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md) for this.
* Also add [#2940](https://github.com/citra-emu/citra/issues/2940) issue. See the example in the guide. It should between `description` and `needs_system_files`.
* 3 screenshots if possible.
* Is that the correct icon for the game?

**Comment by maplesuf** [comment]
177942_thumb

**Comment by Leo626** [comment]
Testcases are no longer added here. You can now submit them through Citra (Help -> Report Compatibility). The icon should named icon.png. Delete the current one and rename the one you added.

File: games/inazuma-eleven-go-light-and-shadow/game.dat [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+title = "0004000000112F00"
+region = "eur"
+release_date = "2014-06-13"
+
+[[releases]]
+title = "0004000000113000"
+region = "eur"
+release_date = "2014-06-13"
+
+[[ testcases ]]
```
Comment (Hexagon12) [inline games/inazuma-eleven-go-light-and-shadow/game.dat]: This line and onward needs to be deleted.

**Comment by Leo626** [comment]
Superseded by #274.

**Pull request closed without merging**